test/suites/standard2/configuration.robot

```diff
@@ -141,37 +141,37 @@ Deploy MicroShift Without CSI Snapshotter
     ...    Remove Storage Drop In Config
     ...    Restart MicroShift
 
-Custom TLS 1_3 configuration
-    [Documentation]    Configure API server to use TLS 1.3 and verify only that
-    ...    version works
-    [Setup]    Setup TLS Configuration    ${TLS_13_MIN_VERSION}
+Custom TLS 1_2 configuration
+    [Documentation]    Configure a custom cipher suite using TLS 1.2 and verify it is used
+    [Setup]    Setup TLS Configuration    ${TLS_12_CUSTOM_CIPHER}
 
-    ${rc}=    Execute Command
-    ...    openssl s_client -connect ${USHIFT_HOST}:6443 -tls1_3 <<< "Q"
-    ...    sudo=True    return_stdout=False    return_stderr=False    return_rc=True
+    ${stdout}    ${rc}=    Openssl Connect Command    -tls1_2 -cipher ECDHE-RSA-CHACHA20-POLY1305
     Should Be Equal As Integers    ${rc}    0
+    Should Contain    ${stdout}    TLSv1.2, Cipher is ECDHE-RSA-CHACHA20-POLY1305
 
-    ${rc}=    Execute Command
-    ...    openssl s_client -connect ${USHIFT_HOST}:6443 -tls1_2
-    ...    sudo=True    return_stdout=False    return_stderr=False    return_rc=True
-    Should Not Be Equal As Integers    ${rc}    0
+    ${stdout}    ${rc}=    Openssl Connect Command    -tls1_2 -cipher ECDHE-RSA-AES128-GCM-SHA256
```

Comment:
Nit: Include a comment on why these two also work?

Reply:
Done in the last commit.